hi-ml-histopathology/src/histopathology/configs/classification/DeepSMILEPanda.py

```diff
@@ -36,26 +31,37 @@
     LoadTilesBatchd,
 )
 
+from histopathology.configs.classification.BaseMIL import BaseMIL
+from histopathology.datasets.panda_dataset import PandaDataset
+from histopathology.models.deepmil import DeepMILModule
+
 
 class DeepSMILEPanda(BaseMIL):
+    """`is_finetune` sets the fine-tuning mode. If this is set, setting cache_mode=CacheMode.NONE takes ~30 min/epoch and
+    cache_mode=CacheMode.MEMORY, precache_location=CacheLocation.CPU takes ~[5-10] min/epoch.
+    Fine-tuning with caching completes using batch_size=4, max_bag_size=1000, num_epochs=20, max_num_gpus=1 on PANDA.
+    """
     def __init__(self, **kwargs: Any) -> None:
         default_kwargs = dict(
             # declared in BaseMIL:
             pooling_type=GatedAttentionLayer.__name__,
             # average number of tiles is 56 for PANDA
             encoding_chunk_size=60,
+            cache_mode=CacheMode.MEMORY,
+            precache_location=CacheLocation.CPU,
+            is_finetune=False,
 
             # declared in DatasetParams:
             local_datasets=[Path("/tmp/datasets/PANDA_tiles"), Path("/tmp/datasets/PANDA")],
-            azure_datasets=["PANDA_tiles", "PANDA"],
+            azure_dataseazure_datasets=["PANDA_tiles", "PANDA"],
             # To mount the dataset instead of downloading in AML, pass --use_dataset_mount in the CLI
             # declared in TrainerParams:
             max_epochs=200,
             # use_mixed_precision = True,
 
             # declared in WorkflowParams:
-            # number_of_cross_validation_splits=5,
-            # cross_validation_split_index=0,
+            number_of_cross_validation_splits=5,
+            cross_validation_split_index=0,
```

Comment:
While you're touching this... it seems there are two calls to the superclass constructor in lines 71 and 72?